src/covid_shared/cli_tools/validation.py

```diff
@@ -1,6 +1,16 @@
+import os
+
+
 def validate_options_with_q(quick: int, mark_best: bool, production_tag: str):
     """
     Quick runs should never be marked 'best' or for production
     """
     if quick and (mark_best or production_tag):
         raise ValueError("Cannot mark a quick snapshot as best or for production.")
+
+
+def ensure_archive_access():
```

Comment:
Both this and `validate_options_with_q` should live in snapshot.  They are not shared utilities.

Reply:
Hmm, we had discussed a long time ago that validate_options_with_q would live in shared_utilities b/c both snapshot and etl (and potentially other pipelines) would use it. I'll go ahead and move `ensure_archive_access` since that's clearly just a snapshot thing